README.md

```diff
@@ -1,3 +1,9 @@
+This repo depends on our [fork of ichord/At.js](https://github.com/CenterForOpenScience/At.js) which
+contains a bug fix for @ mentions. If the PR [ichord/At.js#394](https://github.com/ichord/At.js/pull/394)
+is merged then the original repo will be used instead of our fork.
```

Comment:
Note has been added to the [At.js repo](https://github.com/CenterForOpenScience/At.js). I'll remove the note from the OSF README once this round of review has been completed.